### DISCUSSION TITLE
How can we attribute loaders rsult to a specific arg

### DISCUSSION THREAD

**Discussion by BennaceurHichem** [storybookjs/storybook#15387] [Help]
I'm using a loader in a component story, and the data is correctly fetched, the documentation said that the result of the lozder is accessible from `loaded` object. I tried to pass the `loaded.property` into an arg but an error is shown 
`Cannot find name loaded` which is a typescript error, the data of loaded still not used, and I'm confused on how can I access to the storybook context to see that content of `loaded` to name the debugging easier  
this is my story and loaders where loaded is cannot found : 

`export const Loading = Template.bind({});
(Loading as any).args = {
  options:loaded.options,
//other properties
};
`

`Loading.loaders = [
  async () => ({
    options: (await fetch('https://jsonplaceholder.typicode.com/albums')).json(),
  }),
];`

I want to test the loading of a component with async request, to reflect the loading state, then the reflect the state of the component when data is fully loaded, if I'm wrong in my approach please let me know.

Thanks in advance.

**Comment by shilman** [comment]
Loaders are an experimental feature and they are not yet part of the typescript interface.
